Jumpscale/servers/email_forwarder/JSMailForwarder.py

```diff
@@ -0,0 +1,79 @@
+from .forwarder import MailForwarder
+
+from Jumpscale import j
+
+class JSMailForwarderBase(j.application.JSBaseConfigClass):
+    """Base class that holds the forwarder's configurations as 
+    A JSX schema
+    """
+    
+    _SCHEMATEXT = """
+        @url = jumpscale.mailforwarder.1
+        name* = ""
+        listening_host = ""
+        listening_port = (I)
+        relay_host = ""
+        relay_port = (I)
+        relay_user = ""
+        relay_password = ""
+        relay_ssl = (B)
+        forward_config = (LS)
+    """
+    
+    def _process_forward_config(self):
```

Comment:
I was not able to make the list of objects to work and since there was no docs, I did that work-around. I will check the linked example in the above comment.